spec.bs

```diff
@@ -3586,6 +3621,25 @@ An auction config is a [=struct=] with the following items:
   Restricts the `generateBid()` script's runtime for all buyers without a timeout specified in
   [=auction config/per buyer timeouts=]. If the timeout expires, only the bid submitted via
   `setBid()` is considered.
+: <dfn>per buyer cumulative timeouts</dfn>
+:: Null, a {{Promise}}, failure, or an [=ordered map=] whose [=map/keys=] are [=origins=] and
+  whose [=map/values=] are [=durations=] in milliseconds.
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] are collective timeouts
+  for all interest groups of the buyer represented by the [=map/key=]. Includes the time of loading
+  scripts and signals, and running the `generateBid()` functions. Once the timer expires, the
+  affected buyer's interest groups may no longer generate any bids. All bids generated before the
+  timeout will continue to participate in the auction.
+
+  Note: Implementations <code><span class="allow-2119">should</span></code>
+  attempt, on a best-effort basis, to generate bids for each buyer in priority order, so lower
+  priority interest groups are the ones more likely to be timed out. If promises are passed in to
+  the auction config for fields that support them, the timer for a buyer only starts once all
+  promises blocking that buyer's bidding scripts from running have been resolved.
+
+: <dfn>all buyers cumulative timeout</dfn>
+:: A [=duration=] in milliseconds, initially 50 milliseconds.

```

Comment:
This isn't the case (and limiting a buyer to 50ms will likely break things badly); also this timeout can be turned off, so it probably needs to be duration-or-null, initial to null?

Reply:
Done.